.github/SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a vulnerability
+
+To report a vulnerability, send an email to [cncf-dex-maintainers@lists.cncf.io](mailto:cncf-dex-maintainers@lists.cncf.io)
+detailing the issue and steps to reproduce. The reporter(s) can expect a
+response within 24 hours acknowledging the issue was received. If a response is
```

Comment:
Maybe it is better to take 2 working days for the first response? Vacations and emergencies can slow down the process. I'm just trying to play it safe :)